spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_target_vm_spec.rb

```diff
@@ -66,6 +66,32 @@
     assert_storage(storage, vm)
   end
 
+  context 'vm migration' do
+    before(:each) do
```

Comment:
Sure I can move the ems details into "let" and update those in the context, would that be better?